### ISSUE TITLE
(id) has a corrupt configuration, please destroy the jail

### ISSUE THREAD

**Issue by ghost** [iocage/iocage#377]
Make sure to follow and check these boxes before submitting an issue! Thank you.

- [x] Supply `iocage --version`
0.9.9.1 08/01/2017 (from pkg install py36-iocage

- [x] Supply the commands used, along with any steps to recreate it

```
root@beastie:/usr/home/voltagex # iocage list
********************************************************************************
fdescfs(5) is not mounted, performance  may suffer. Please run:
mount -t fdescfs fdesc /dev/fd
You can also permanently mount it in /etc/fstab with the following entry:
fdesc /dev/fd  fdescfs  rw  0  0
********************************************************************************

84122687-a27d-11e6-b259-0cc47a6bbf34 has a corrupt configuration,please destroy the jail.

```

- [x] Provide the output from the command you issued.
- [x] Supply what you expected the result or output to be
jail upgrade OR instructions on how to delete the jail
- [ ] Checked that the problem has not already been fixed on `master` if using
 a stable release.

How would I upgrade to master without messing up the pkg system or risking my existing jails?

**Comment by skarekrow** [comment]
Do a backup before playing with iocage. It’s a beta so things can and do happen, like that bug. 

I suggest deleting that jails config.json if it was a legacy jail, and pkg upgrading to the latest to let it redo the migration step. @araujobsd has pushed it into ports, so it will be in pkg form soon :)

This bug *should* be fixed in 0.9.9.2. I’ll mark this as needs more information in the meantime to remind myself to check back with you to see.

**Comment by ghost** [comment]
Right yeah. I think I caused this by not making sure all jails were stopped before upgrading iocage. I've backed up the data and zfs destroyed the original jail. Just waiting for Cython install to complete and I'll try creating a new jail on git master.
